vagga.yaml

```diff
@@ -2,19 +2,41 @@ containers:
   pip-freeze:
     setup:
       - !Alpine v3.4
+      - !BuildDeps
+        - postgresql-dev
```

Comment:
Documentations says that postgresql needed for building psycopg, libpq - needed for it in runtime.
There's no other info.
But I think difference is also in !BuildDeps and !Install